libpod/runtime_migrate.go

```diff
@@ -0,0 +1,43 @@
+package libpod
+
+import (
+	"context"
+	"path/filepath"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+func (r *Runtime) migrate() error {
+	runningContainers, err := r.GetRunningContainers()
+	if err != nil {
+		return err
+	}
+
+	allCtrs, err := r.state.AllContainers()
+	if err != nil {
+		return err
+	}
+
+	logrus.Infof("stopping all containers")
+	for _, ctr := range runningContainers {
+		logrus.Infof("stopping %s", ctr.ID())
+		ctr.Stop()
+	}
+
+	for _, ctr := range allCtrs {
+		if strings.HasPrefix(ctr.config.ConmonPidFile, ctr.state.RunDir) {
+			logrus.Infof("changing conmon PID file for %s", ctr.ID())
+			ctr.config.ConmonPidFile = filepath.Join(ctr.config.StaticDir, "conmon.pid")
```

Comment:
I really wish we had a way to detect user-specified commands here, so we wouldn't rewrite for people who had explicitly specified that path...